website/integrations/services/omni/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Integrate with Omni
+sidebar_label: Omni
+support_level: community
+---
+
+## What is Omni
+
+> Omni manages Kubernetes on bare metal, virtual machines, or in a cloud.
+>
+> -- https://github.com/siderolabs/omni
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `omni.company` is the FQDN of the Omni installation.
+- `authentik.company` is the FQDN of the authentik installation.
+
+:::note
+This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
+:::
+
+## authentik configuration
+
+To support the integration of _Omni_ with authentik, you need to create a property mapping and application/provider pair in authentik.
+
+### Create a Property Mapping, Application and Provider in authentik
+
+1. Log in to authentik as an admin, and open the authentik Admin interface.
+2. Navigate to **Cusotmization** > **Property Mappings** and click **Create** to create a property mapping.
+
+- **Choose a Property Mapping type**: Select SAML Provider Property Mapping as the property mapping type.
+
+- **Configure the Property Mapping**:
+    - Name: `<property_mapping_name>` (e.g. `Omni Mapping`)
```

Comment:
```suggestion
    - **Name**: `<property_mapping_name>` (e.g. `Omni Mapping`)
```

Reply:
Per the Style Guide, we [ideally] bold the UI labels. BUt it can get quite boring to do when there is a long list...